lib/AdobeState.js

```diff
@@ -159,18 +173,13 @@ class AdobeState {
    * Creates a request url.
    *
    * @private
-   * @param {string} key the key of the state store
+   * @param {string} containerURLPath defaults to '' to hit the container
+   *   endpoint, add /data/key to hit the key endpoint
    * @param {object} queryObject the query variables to send
    * @returns {string} the constructed request url
    */
-  createRequestUrl (key, queryObject = {}) {
-    let urlString
-
-    if (key) {
-      urlString = `${this.endpoint}/${API_VERSION}/containers/${this.namespace}/data/${key}`
-    } else {
-      urlString = `${this.endpoint}/${API_VERSION}/containers/${this.namespace}`
-    }
+  createRequestUrl (containerURLPath = '', queryObject = {}) {
+    const urlString = `${this.endpoint}/${API_VERSION}/containers/${this.namespace}${containerURLPath}`
```

Comment:
reviewer note: API_VERSION will be removed in https://github.com/adobe/aio-lib-state/pull/165